### DISCUSSION TITLE
Capacity of Filecoin with current parameters? (PoSt frequency, gas cost, etc)

### DISCUSSION THREAD

**Discussion by The-Wayvy** [filecoin-project/FIPs#739] [General]
Hello

I calculated the storage capacity of Filecoin to be 64 Petabytes, much less than the claimed 1 Exabyte of active deals.

What am I not understanding about the cost of proving storage?

Given: 
~ the average gas cost per PoSt is 50 million
~ each sector is no more than 64GB 
~ one PoSt is required every 24 hours
~ 50 trillion gas worth of computation is handled by the network every 24 hours

1 million PoSt per day : (5 * 10^13 gas) / (5 * 10^7 gas per PoSt)  
64 Petabytes of storage proven each day: 10^6 PoSt * 64 GB per PoSt 

That's a 15x difference vs. the 1.1 Exabyte total that I see on Block Explorers

https://filfox.info/en/stats/gas
https://dashboard.starboard.ventures/transactions-usage#gas-usage

**Comment by Stebalien** [comment]
It's one PoSt proof per 10 partitions (max) where each partition can have 2300 sectors (max). Although it's usually not quite _that_ efficient.
